website/docs/guides/dbt-python-bigframes.md

```diff
@@ -0,0 +1,384 @@
+
+Then enable the BigQuery API which also enables the following additional APIs automatically
+[BigQuery API's](https://cloud.google.com/bigquery/docs/enable-assets#automatic-api-enablement)
+
+
+2. **Create a service account and grant IAM permissions**
+
+This service account will be used by dbt to read and write data on BigQuery and use BigQuery Studio Notebooks
+
+```python
+#Create Service Account
+gcloud iam service-accounts create dbt-bigframes-sa
+#Grant BigQuery User Role
+gcloud projects add-iam-policy-binding ${GOOGLE_CLOUD_PROJECT} --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/bigquery.user
+#Grant BigQuery Data Editor role. This can be restricted at dataset level
+gcloud projects add-iam-policy-binding ${GOOGLE_CLOUD_PROJECT} --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/bigquery.dataEditor
+#Grant Service Account user 
+gcloud projects add-iam-policy-binding ${GOOGLE_CLOUD_PROJECT} --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/iam.serviceAccountUser
+#Grant Colab Entperprise User
+gcloud projects add-iam-policy-binding ${GOOGLE_CLOUD_PROJECT} --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/aiplatform.colabEnterpriseUser
+```
+
+NOTE: There are other optional IAM requirements required to apply for using remote functions and ML remote models.
+
+[Remote Functions](https://cloud.google.com/bigquery/docs/use-bigquery-dataframes#remote-function-requirements)
+
+[ML Remote Models](https://cloud.google.com/bigquery/docs/use-bigquery-dataframes#remote-models)
+
+3. *(Optional)* **Create a test BigQuery Dataset**
+
+Create a new BigQuery Dataset if you don't already have one.
+
+```python
+#Create BQ dataset 
+bq mk --location=${REGION} echo "${GOOGLE_CLOUD_PROJECT}" | tr '-' '_'_dataset
+```
+
+4. *(Optional)* **Create a GCS bucket to stage the python code**
+
+If you wish to store the python compiled code on a GCS bucket, create a new one.
+
+```python
+#Create GCS bucket
+gcloud storage buckets create gs://${GOOGLE_CLOUD_PROJECT}-bucket --location=${REGION}
+#Grant Storage Admin over the bucket to your SA 
+xgcloud storage buckets add-iam-policy-binding gs://${GOOGLE_CLOUD_PROJECT}-bucket --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/storage.admin
+```
+
+5. **Create a GCS bucket to hold the logs**
+```python
+#Create GCS bucket
+gcloud storage buckets create gs://${GOOGLE_CLOUD_PROJECT}-bucket-logs --location=${REGION}
+#Grant Storage Admin over the bucket to your SA 
+gcloud storage buckets add-iam-policy-binding gs://${GOOGLE_CLOUD_PROJECT}-bucket-logs --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/storage.admin
+```
+
+## Client Side Configuration 
```

Comment:
Please remove the section about dbt core setup. The requirements from the beginning is already based on dbt Cloud and they should have this setup before getting started. You can also point them to the BQ + dbt quickstart as an alternative.